apps/aecore/test/aec_pow_tests.erl

```diff
@@ -169,22 +170,69 @@ conversion_test_() ->
      ]
     }.
 
-recalc_test_() ->
+target_adj_test_() ->
     {setup,
-     fun setup/0,
-     fun teardown/1,
-     [{"Block building progressing twice as slow as expected, double target to make mining easier",
-       fun() ->
-               ExpectedTimeDiffBetweenBlocks = 500,
-               CurrentTimeDiffBetweenBlocks = 1000,
-               CurrentTarget = ?TEST_MODULE:integer_to_scientific(16#400000),
-               ExpectedTarget = ?TEST_MODULE:integer_to_scientific(16#800000),
-               ?assertEqual(ExpectedTarget,
-                            ?TEST_MODULE:recalculate_target(
-                               CurrentTarget, ExpectedTimeDiffBetweenBlocks, CurrentTimeDiffBetweenBlocks))
-       end}
-     ]
-    }.
+     fun setup_target/0,
+     fun teardown_target/1,
+     [{"With constant PoW capacity the target will stabilize (seed = " ++ integer_to_list(S) ++ ")",
+      fun() ->
+          Seed = {1, 1, S},
+          rand:seed(exs1024s, Seed),
+          PoWCapacity = 100,
+          TargetSpeed = 1 / 5, %% 1 block per 5 minutes
+          ExpectedDifficulty = PoWCapacity / TargetSpeed,
+          InitBlocks = [aec_block_genesis:genesis_block()],
+          Chain = [Top | _] = mine_chain(InitBlocks, 100, PoWCapacity),
+          Difficulties = [ aec_blocks:difficulty(B) || B <- Chain ],
+          %% ?debugFmt("Difficulties: ~p", [Difficulties]),
+          Window = 20,
+          AvgDiffWindow = lists:sum(lists:sublist(Difficulties, Window)) / Window,
+          RateWindow = 50,
```

Comment:
It looks like choice of this value has implicit assumption on aec_governance:blocks_to_check_difficulty_count(). Please add assert.

Reply:
Not really... This is more related to the 100 blocks we use in the simulation.